corehq/messaging/scheduling/static/scheduling/js/conditional_alert_main.js

```diff
@@ -34,5 +34,34 @@ hqDefine("scheduling/js/conditional_alert_main", [
         // setup tab
         basicInformation.setRuleTabVisibility();
         $("#conditional-alert-basic-info-panel").koApplyBindings(basicInformation);
+
+        $("#conditional-alert-save-btn").click(function () {
+            var newRuleName = $("#id_conditional-alert-name").val();
+            if (!name) {
+                $("#conditional-alert-form").submit();
+            } else if (name !== newRuleName) {
+                var caseType = $(
+                    "#select2-id_criteria-case_type-container"
+                ).text();
+                var caseCountUrl = $("#conditional-alert-case-count-url").attr(
+                    "url"
+                );
+                $.ajax({
+                    type: "GET",
+                    url: caseCountUrl,
+                    data: { case_type: caseType },
+                    success: function (data) {
```

Comment:
An error function would be good. Errors aren't likely here, but if ES is having problems, they could happen. If you only open the popup on success, the usual way we handle errors is to change the text of the button to say something like "Try Again."